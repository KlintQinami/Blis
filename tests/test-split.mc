void print_string_list(u8[][] strings)
{
  int i;
  for (i = 0; i < length(strings); i = i + 1) {
    print(strings[i]);
    print("\n");
  }
}

int main()
{
  print_string_list(split("aababba", 'a'));
  print("\n");
  print_string_list(split("babb", 'a'));
  print("\n");
  print_string_list(split("abbbbbbbba", 'a'));

  return 0;
}
