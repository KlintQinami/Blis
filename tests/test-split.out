b
bb

b
bb

bbbbbbbb
